src/api.jl

```diff
@@ -348,12 +348,12 @@ function Minuit(fcn, x0...; grad=nothing, error=(), errordef=1.0, names=(), limi
         #---Get the limits, first if the user provided them, then from the keyword arguments----------------
         limit = i > length(limits) ? haskey(kwargs, Symbol("limit_", name)) ? kwargs[Symbol("limit_", name)] : nothing : limits[i]
         if limit !== nothing
-            if limit[1] == -Inf
```

Comment:
this used to have a "bug" where if the limits are `(-Inf, Inf)` it's going to set `UpperLimit` to `Inf` and causing results to be `NaN`